semgrep-core/tests/tainting_rules/go/token_labels.go

```diff
@@ -0,0 +1,8 @@
+// See https://github.com/returntocorp/semgrep/issues/6355 if this breaks.
```

Comment:
The `tainting_rules` folder comes from a time when you couldn't pass taint rules to `-rules`, you had to use a different switch `-tainting_rules` or something. Nowadays I just add these tests in `tests/rules` but prefixed with `taint_`. I'll see if I can move all tests soon.